setup.py

```diff
@@ -26,7 +26,7 @@
 install_extras = {
     'client': [
         'gevent>=1.3.0',
-        'protobuf>=3.0.0',
+        'protobuf>=3.0.0,<3.18.0',
```

Comment:
version spec needs to be updated here too

Reply:
Done!